mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/CheckCmd.scala

```diff
@@ -70,37 +71,43 @@ class CheckCmd(name: String = "check", description: String = "Check a TLA+ speci
     opt[Boolean](name = "output-traces", description = "save an example trace for each symbolic run, default: false",
         default = false)
 
-  def collectTuningOptions(): Map[String, String] = {
-    val tuning = tuningOptionsFile.map(f => loadProperties(f)).getOrElse(Map())
-    overrideProperties(tuning, tuningOptions.getOrElse("")) ++ Map("search.outputTraces" -> saveRuns.toString)
+  override def toConfig(): Try[Config.ApalacheConfig] = {
+    val loadedTuningOptions = tuningOptionsFile.map(f => loadProperties(f)).getOrElse(Map())
+    val combinedTuningOptions = overrideProperties(loadedTuningOptions, tuningOptions.getOrElse("")) ++ Map(
+        "search.outputTraces" -> saveRuns.toString)
+
+    super.toConfig().map { cfg =>
+      val newCfg = cfg.copy(
+          checker = cfg.checker.copy(
+              nworkers = nworkers,
```

Comment:
a question outside of this PR, shall we remove the `nworkers` parameters after all, as we do not have any plans to integrate the multicore branch in the near future?

Reply:
Sure! I’ll do as a follow up.